site/README.md

```diff
@@ -0,0 +1,68 @@
+## Basic manual for website editing
+
+### Edit or add documentation pages
+
+To edit and/or add documentation, you need to have a [GitHub](https://github.com/login) account.
+To change documentation files or add a documentation page,
+simply click `Edit this page` on the page you would like to edit.
+If you need to add a child page, click `Create child page`.
+
+If you need to edit the text that has the markup [markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet),
+click on the `Fork this reposytory` button.
+
+Read how to edit files for githab ([GitHub docs](https://docs.github.com/en/github/managing-files-in-a-repository/editing-files-in-another-users-repository)).
```

Comment:
Please check text using some special tools. For example, `githab` is a wrong word.